stories/core/Header.stories.tsx

```diff
@@ -0,0 +1,215 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Meta, Story } from '@storybook/react/types-6-0';
```

Comment:
Can import from '@storybook/react'

Reply:
Updated (HorizontalTab have it this way too, with the `types-6-0` part)